packages/maskbook/src/plugins/RedPacket/UI/RedPacketInList.tsx

```diff
@@ -25,6 +37,60 @@ const useStyles = makeStyles((theme: Theme) =>
             fontSize: 12,
             color: theme.palette.text.secondary,
         },
+        root: {
+            borderRadius: 10,
+            border: `solid 1px ${theme.palette.divider}`,
+            marginBottom: theme.spacing(1.5),
+            position: 'static !important' as any,
+            height: 'auto !important',
+            padding: theme.spacing(2),
+        },
+        box: {
+            display: 'flex',
+            width: '100%',
+        },
+        content: {
+            transform: 'translateY(-4px)',
+            width: '100%',
+            padding: theme.spacing(0, '1rem'),
```

Comment:
then why use `theme.spacing`? Another thing, why you'd want to use `rem`? I'm curious how it work with shadow root

Reply:
1rem = 16px in shadow root as well. But anyway let me change this to '0 16px'.